### PR TITLE
DEVDOCS-4429: [new] Headless Carts, add locale info; [update] Customer Login

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4429]

## What changed?
Added note and request body examples for the managing carts article.

## Anything else?

[DEVDOCS-4429]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kristinapototska — DISMISSED** [review]
LGTM 👍

File: docs/api-docs/headless/carts.md [hunk 1]
```diff
@@ -28,7 +32,28 @@ X-Auth-Token: {{ACCESS_TOKEN}}
             "product_id": 80,
             "variant_id": 64
         }
-    ]
+    ],
+    "locale": "en-us"
+}
+```
+To create a cart with option selections, include `option_id` and `option_value` in your `POST` request.
+```json
```
Comment (bc-andreadao) [inline docs/api-docs/headless/carts.md:40]:
```suggestion
```json lineNumbers
```

Comment (bc-andreadao) [reply docs/api-docs/headless/carts.md:40]: @bc-traciporter I noticed that the other code blocks don't have "lineNumbers," so if you commit my suggestion, there are others also that need to be changed

**Review by bc-andreadao — COMMENTED** [review]
One thing I noticed was that the code blocks throughout the document could use consistent white space (e.g. 2 indents)

File: docs/api-docs/headless/carts.md (hunk 1 above)
Comment (bc-traciporter) [reply docs/api-docs/headless/carts.md:40]: @bc-andreadao what does lineNumbers do? Can you tell me why it should be added?

Comment (bc-andreadao) [reply docs/api-docs/headless/carts.md:40]: It adds lineNumbers for your code blocks

**Comment by bc-traciporter** [comment]
@bc-andreadao I added lineNumbers

**Review by bc-andreadao — APPROVED** [review]
Looks good, I fixed the white space for the code blocks (2 indents like our newer docs)

File: docs/api-docs/headless/carts.md [hunk 2]
```diff
@@ -89,20 +118,25 @@ The response will contain `cart_url` and `checkout_url` parameters - use these U
...
 X-Auth-Token: {{ACCESS_TOKEN}}
+
+{
+  // request body
+}
+
 ```
 
-### Logging in and redirecting a customer
+### Signing in and redirecting a customer
```
Comment (slsriehl) [inline docs/api-docs/headless/carts.md:134]: @bc-traciporter Let's revise this section to point to our dedicated documentation on the Customer Login API.

**Review by slsriehl — CHANGES_REQUESTED** [review]

File: docs/api-docs/headless/carts.md [hunk 3]
```diff
@@ -4,80 +4,112 @@
 
 In this section, we will explain how to use the Carts API to create and manage carts. Additionally, we will discuss how to redirect shoppers from a headless storefront to the BigCommerce hosted cart and checkout pages.
 
+<!-- theme: info -->
+> #### Note
+> The Carts API supports selling in different countries by allowing overrides for locale and product details. Merchants can supply a cart's locale, alternative product names, and option values.
```
Comment (slsriehl) [inline docs/api-docs/headless/carts.md:9]:
```suggestion
> The Carts API supports selling in different markets by allowing locale-based overrides. To use alternative product names and option values, <specify those values on the product object and then add the corresponding locale to a cart>.  
```

@bc-traciporter I've added the carrots around approximate text that captures this gist. I leave the actual writing to you, as well the fact checking if this seems wrong or doesn't make sense. I am not 100% about whether option values are pre-existing on the product object, or defined ad hoc in the cart. If they're ad hoc, the gist could be something like this:

```suggestion
> The Carts API supports selling in different markets by allowing locale-based overrides. To use alternative product details, <use the product object to specify product info for a given locale, then add the corresponding locale to a cart. Your application can use the locale to set its own option values>.  
```

File: docs/api-docs/headless/carts.md [hunk 4]
```diff
@@ -4,80 +4,112 @@
 
 In this section, we will explain how to use the Carts API to create and manage carts. Additionally, we will discuss how to redirect shoppers from a headless storefront to the BigCommerce hosted cart and checkout pages.
 
+<!-- theme: info -->
+> #### Note
```
Comment (slsriehl) [inline docs/api-docs/headless/carts.md:8]:
```suggestion
> #### Locale support
```

File: docs/api-docs/headless/carts.md (hunk 3 above)
Comment (slsriehl) [reply docs/api-docs/headless/carts.md:9]: never mind, this is wrong!

**Comment by slsriehl** [comment]
further feedback forthcoming in slack.

File: docs/api-docs/headless/carts.md [hunk 5]
```diff
@@ -4,24 +4,24 @@
...
+<!-- theme: info -->
+> #### Locale support
+> The Carts API supports selling in different markets by allowing locale-based overrides for product details. Supply a cart's locale and add `option_selections` to the cart's line items to configure alternative product names, option names, and modifier values. The Carts API stores the locale and makes it available to the Checkout and Orders APIs. 
+
 ## Creating a cart
 
 The [Carts API](/api-reference/store-management/carts) lets you create carts for both existing and guest customers. 
 
-To create a cart, send a `POST` request to the [Create a Cart](/api-reference/store-management/carts/cart/createacart) endpoint.
+To create a cart, send a request to the [Create a Cart](/api-reference/store-management/carts/cart/createacart) endpoint.
```
Comment (slsriehl) [inline docs/api-docs/headless/carts.md:15]:
```suggestion
To create a cart, send a request to the [Create a cart](/api-reference/store-management/carts/cart/createacart) endpoint.
```

Comment (slsriehl) [reply docs/api-docs/headless/carts.md]: done locally

**Review by slsriehl — APPROVED** [review]
releasing it into the wild, thanks for your patience @bc-traciporter . Please open another PR if you want things to be different

**Pull request merged**